### ISSUE TITLE
Missing ARIA roles in fast-tooling navigation menu

### ISSUE THREAD

**Issue by chrisdholt** [microsoft/fast#4490]
The following bug was found during an automated accessibility scan using Accessibility Insights for web.

**ARIA Required Parent: Ensures elements with an ARIA role that require parent roles are contained by them**
The Navigation Menu component includes items with a role of `menuitem` but is missing a required parent (menu or menubar respectively) for those roles.

**What behavior did you expect?**
When items have a role of "menuitem", the Navigation Menu should have a role of "menu".
